### PR TITLE
Add pages from appliance to k8s for datahub and databroker

### PR DESCRIPTION
**Pull request by TzviLipschitz**
https://softwareag-c8y-docs-build-pr-2070.surge.sh/

### CODE REVIEW

File: content/edge-kubernetes/edge-configuration-bundle/configuring-apama.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
```
Comment (TzviLipschitz) [inline content/edge-kubernetes/edge-configuration-bundle/configuring-apama.md:1]: rework

File: content/edge-kubernetes/edge-configuration-bundle/configuring-karaf.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
+---
```
Comment (TzviLipschitz) [inline content/edge-kubernetes/edge-configuration-bundle/configuring-karaf.md:1]: rework

File: content/edge-kubernetes/edge-configuration-bundle/configuring-opcua.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
```
Comment (TzviLipschitz) [inline content/edge-kubernetes/edge-configuration-bundle/configuring-opcua.md:1]: rework

File: content/edge-kubernetes/edge-databroker-bundle/databroker-edge-uninstall.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
+---
```
Comment (TzviLipschitz) [inline content/edge-kubernetes/edge-databroker-bundle/databroker-edge-uninstall.md:1]: remove

File: content/edge-kubernetes/edge-working-with-datahub-bundle/datahub-edge.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+weight: 10
+title: Analytical querying with DataHub Edge
+layout: bundle
+section:
+  - edge_server
+---
+
+DataHub Edge is an optional component of Edge. DataHub Edge complements the ad-hoc querying of recent device data with analytical querying over long periods of time. For that purpose, data is moved from the Operational Store of Edge to a local data lake, with the data being stored in a concise and query-efficient format. DataHub Edge allows you to run SQL queries against the data lake contents so that you can gain more insights into your device data.
+
+DataHub Edge is the counterpart of DataHub, the variant for cloud deployments. To learn more about DataHub in general, see [DataHub overview](/datahub/datahub-overview). To learn more about DataHub Edge, see [Running DataHub on the Edge](/datahub/running-datahub-on-the-edge/).
```
Comment (TzviLipschitz) [inline content/edge-kubernetes/edge-working-with-datahub-bundle/datahub-edge.md:11]: update links

File: content/edge-kubernetes/using-edge-bundle/branding.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+<img src="/images/users-guide/enterprise-tenant/et-branding.png" alt="Branding tab">
+
+{{< c8y-admon-info >}}
+Use the domain name to access the Edge platform to properly apply the branding and see the effects after branding.
+{{< /c8y-admon-info >}}
+
+To access the branding feature:
+
+1. Log in to the {{< management-tenant >}} using the Edge administrator credentials created during the installation.
```
Comment (TzviLipschitz) [inline content/edge-kubernetes/using-edge-bundle/branding.md:21]: using management domain (i.e management.myown.iot.com) and administrator credentials created during the installation.

File: content/edge-kubernetes/k8-edge-connecting-edge-to-cloud-bundle/data-exchange-using-data-broker.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+2. In your Edge, go to the Administration application. Click **Data broker** > **Data connectors**.
+
+   <img src="/images/users-guide/enterprise-tenant/et-data-broker-navigator.png" alt="Data broker menus">
+
+3. Click **Add data connector** and provide all the information and filters. See [To add a data connector](/data-broker/data-broker-application/#to-add-a-data-connector).
+
+   Note down the security code. This security code will be used to subscribe the connector in the tenant account.
+
+4. Log in to the {{< product-c8y-iot >}} tenant account.
```
Comment (TzviLipschitz) [inline content/edge-kubernetes/k8-edge-connecting-edge-to-cloud-bundle/data-exchange-using-data-broker.md:43]: link to accessing edge where login is explained

File: content/edge-kubernetes/using-edge-bundle/branding.md (hunk 6 above)
Comment (TzviLipschitz) [reply content/edge-kubernetes/using-edge-bundle/branding.md:21]: link to accessing edge where login is explained

File: content/edge-kubernetes/edge-configuration-bundle/configuring-email-server.md [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+  - edge_server
+---
+
+Configuring an email server enables you to receive email notifications about events, alarms, and also to reset your password. In case you forget the password, the Edge appliance mails you the password reset link to reset your password.
+
+To configure the "reset password" template and email server settings, perform the following steps:
+
+1. Log into the {{< management-tenant >}} using *https://&#60;tenant-domain>/apps/administration/index.html#/configuration*.
+
+	- Username: management/<*Edge admin username*>
```
Comment (TzviLipschitz) [inline content/edge-kubernetes/edge-configuration-bundle/configuring-email-server.md:15]: link to accessing edge

File: content/edge-kubernetes/connecting-devices-to-edge-bundle/connecting-modbus-device.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
+---
```
Comment (TzviLipschitz) [inline content/edge-kubernetes/connecting-devices-to-edge-bundle/connecting-modbus-device.md:1]: remove

File: content/edge-kubernetes/connecting-devices-to-edge-bundle/connecting-devices.md [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+layout: bundle
+section:
+  - edge_server
+---
+
+Edge uses REST and MQTT as protocols for external communication. Both protocols may be used to interface devices with Edge. For more information, see [Device integration using REST](/device-integration/device-integration-rest/) and [SmartREST 2.0](/smartrest/smartrest-two/).
+
+Additionally, Edge offers:
+
+* Cloud Fieldbus functionality to collect data from fieldbus devices and remotely manage them. For example, Modbus protocol.
```
Comment (TzviLipschitz) [inline content/edge-kubernetes/connecting-devices-to-edge-bundle/connecting-devices.md:13]: If your use case involves Modbus-TCP or Modbus-RTU, we recommend migrating to the thin-edge.io connectors for Cumulocity IoT. For users of CANopen, we suggest considering migration to one of our partner-supported solutions or deploying Cumulocity Linux Agent as an additional component.

Comment (TzviLipschitz) [reply content/edge-kubernetes/connecting-devices-to-edge-bundle/connecting-devices.md:13]: move to second point

File: content/edge-kubernetes/edge-configuration-bundle/updating-ssl-certificate.md [hunk 11]
```diff
@@ -0,0 +1,31 @@
+---
```
Comment (TzviLipschitz) [inline content/edge-kubernetes/edge-configuration-bundle/updating-ssl-certificate.md:1]: rework

**Comment by TzviLipschitz** [comment]
Added preview: https://softwareag-c8y-docs-build-pr-2070.surge.sh/

File: content/edge-kubernetes/k8-edge-connecting-edge-to-cloud-bundle/data-exchange-using-data-broker.md [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+
+   Note down the security code. This security code will be used to subscribe the connector in the tenant account.
+
+4. Log in to the {{< product-c8y-iot >}} tenant account. For more information on, see [To access Edge](/edge-kubernetes/installing-edge-on-k8/#to-access-cumulocity-iot-edge)
+
+5. In the tenant account, go to the Administration application. Click **Data broker** > **Data subscriptions** to subscribe the connector created in your Edge appliance.
+
+   Click **Add data subscription** and provide the security code. Click **Submit** and accept the subscription. See [Data subscriptions](/data-broker/data-broker-application/#data-subscriptions).
+
+   You can now navigate to the Device Management application or the Cockpit application. You will find a new “virtual group” with a specific icon showing the forwarded devices. The group will have the same name as your subscription.
```
Comment (MWindel) [inline content/edge-kubernetes/k8-edge-connecting-edge-to-cloud-bundle/data-exchange-using-data-broker.md:47]:
```suggestion
   You can now navigate to the Device Management application or the Cockpit application. You will find a new "virtual group" with a specific icon showing the forwarded devices. The group will have the same name as your subscription.
```

File: content/edge-kubernetes/k8-using-edge-bundle/developing-an-application.md [hunk 13]
```diff
@@ -0,0 +1,31 @@
...
+weight: 85
+title: Developing Cumulocity IoT web applications
+layout: redirect
+---
+
+If you develop a {{< product-c8y-iot >}} web application using the [Web SDK](/web/introduction/), for an Edge appliance configured with a certificate not trusted by Node.js (for example, a self-signed certificate), then you must ensure that Node.js trusts the root certificate.
+
+To add the Edge appliance's self-signed certificate to the Node.js trust store, set the environment variable `NODE_EXTRA_CA_CERTS` to the path of the certificate before executing the `npm` commands.
+
+For example:
```
Comment (MWindel) [inline content/edge-kubernetes/k8-using-edge-bundle/developing-an-application.md:11]:
```suggestion
**Example:**
```

File: content/edge-kubernetes/k8-using-edge-bundle/developing-an-application.md [hunk 14]
```diff
@@ -0,0 +1,31 @@
...
+
+On Windows Powershell
```
Comment (MWindel) [inline content/edge-kubernetes/k8-using-edge-bundle/developing-an-application.md:13]:
```suggestion
On Windows Powershell:
```

File: content/edge-kubernetes/k8-using-edge-bundle/developing-an-application.md [hunk 15]
```diff
@@ -0,0 +1,31 @@
...
+
+```shell
+$env:NODE_EXTRA_CA_CERTS=<path-to-the-certificate>\certificate.pem
+```
+
+On Windows command prompt
```
Comment (MWindel) [inline content/edge-kubernetes/k8-using-edge-bundle/developing-an-application.md:19]:
```suggestion
On Windows command prompt:
```

File: content/edge-kubernetes/k8-using-edge-bundle/developing-an-application.md [hunk 16]
```diff
@@ -0,0 +1,31 @@
...
+
+```shell
+set NODE_EXTRA_CA_CERTS=<path-to-the-certificate>\certificate.pem
+```
+
+On Linux platforms
```
Comment (MWindel) [inline content/edge-kubernetes/k8-using-edge-bundle/developing-an-application.md:25]:
```suggestion
On Linux platforms:
```

File: content/edge/edge-connectivity-bundle/data-exchange-using-data-broker.md [hunk 17]
```diff
@@ -30,12 +30,10 @@ A data connector describes the subset of the data that you would like to send to
 
 To create a data connector and upload the data to the tenant account, perform the following steps:
 
-1. In your Edge applaince, log in to the *edge* tenant.
+1. In your Edge appliance, log in to the *edge* tenant.
```
Comment (MWindel) [inline content/edge/edge-connectivity-bundle/data-exchange-using-data-broker.md:33]:
```suggestion
1. In your Edge appliance, log in to the "edge" tenant.
```

File: content/edge/using-edge-bundle/branding.md [hunk 18]
```diff
@@ -4,27 +4,12 @@ title: Branding
...
-
-	- Username: management/<*Edge admin username*>
-	- Password: password provided during the installation
-
-2. Switch to the Administration application using the application switcher at the right of the top bar **<img class="Default" src="/images/icons/switcher-icon.png" alt="Application switcher" style="display: inline; float: none">**.
-
-3. Go to **Settings** > **{{< enterprise-tenant >}}**.
-
-For information about branding configuration, see [Branding](/enterprise-tenant/customization/#branding).
+Go to **Settings** > **{{< enterprise-tenant >}}** and follow the instructions detailed in the [Branding](enterprise-tenant/customization/#branding) section.
```
Comment (MWindel) [inline content/edge/using-edge-bundle/branding.md:15]:
```suggestion
Go to **Settings** > **{{< enterprise-tenant >}}** and follow the instructions detailed in [Branding](enterprise-tenant/customization/#branding).
```

File: content/edge-kubernetes/k8-using-edge-bundle/branding.md [hunk 19]
```diff
@@ -0,0 +1,13 @@
...
+layout: redirect
+---
+
+With the Branding feature, you can fully customize the look of your tenant to your own preferences. To access the branding feature:
+
+Log into the {{< management-tenant >}} using the management domain (for example, management.myown.iot.com) and administrator credentials created during the installation. For more information on logging in to the {{< management-tenant >}}, see [To access Edge](/edge-kubernetes/installing-edge-on-k8/#to-access-cumulocity-iot-edge)
+
+Switch to the Administration application using the application switcher at the right of the top bar **<img class="Default" src="/images/icons/switcher-icon.png" alt="Application switcher" style="display: inline; float: none">**.
+
+Go to **Settings** > **{{< enterprise-tenant >}}** and follow the instructions detailed in the [Branding](enterprise-tenant/customization/#branding) section.
```
Comment (MWindel) [inline content/edge-kubernetes/k8-using-edge-bundle/branding.md:13]:
```suggestion
Go to **Settings** > **{{< enterprise-tenant >}}** and follow the instructions detailed in [Branding](enterprise-tenant/customization/#branding).
```

File: content/edge-kubernetes/k8-edge-configuration-bundle/configuring-email-server.md [hunk 20]
```diff
@@ -0,0 +1,15 @@
...
+weight: 65
+title: Configuring the email server
+layout: bundle
+section:
+  - edge_server
+---
+
+Configuring an email server enables you to receive email notifications about events, alarms, and also to reset your password. In case you forget the password, the Edge appliance mails you the password reset link to reset your password.
+
+To configure the "reset password" template and email server settings, perform the following steps:
```
[the remark this answers is not in the document]
Comment (TzviLipschitz) [reply content/edge-kubernetes/k8-edge-configuration-bundle/configuring-email-server.md:11]: @MWindel To get to the reset password template you go to Settings > Enterprise tenant using the left hand navigation bar and the Configuration in the top navigation bar and then there is a scrollable page with various options including email server and password reset. Reset password template refers to a text section where you can configure what the email sent will say.

**Review by MWindel — APPROVED** [review]

**Review by bbyreddy — APPROVED** [review]

**Pull request merged**